Common/Data/UniverseSelection/FuturesChainUniverse.cs

```diff
@@ -32,31 +31,50 @@ public class FuturesChainUniverse : Universe
     {
         private static readonly IReadOnlyList<TickType> dataTypes = new[] { TickType.Quote, TickType.Trade, TickType.OpenInterest };
 
-        private readonly Future _future;
         private readonly UniverseSettings _universeSettings;
         private SubscriptionManager _subscriptionManager;
-
-        private HashSet<Symbol> _cachedSelection;
         private DateTime _cacheDate;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FuturesChainUniverse"/> class
+        /// </summary>
+        /// <param name="future">The canonical future chain security</param>
+        /// <param name="universeSettings">The universe settings to be used for new subscriptions</param>
+        /// <param name="subscriptionManager">The subscription manager used to return available data types</param>
+        public FuturesChainUniverse(Future future,
+            UniverseSettings universeSettings,
+            SubscriptionManager subscriptionManager)
```

Comment:
is this `SubscriptionManager` required here? it seems out of place? Perhaps we only need certain functions available via an interface?

Reply:
I'm seeing that the previous ctor required this as well -- I think it's worth checking to see if this class only requires a subset and perhaps another PR we can modify this class to depend on an interface which provides the behavior -- this is all in an effort to remove algorithm layer objects as dependencies to various lean engine components, such as universes.

Also interesting that the options chain universe doesn't require `SubscriptionManager` -- I would have expected them to have the same dependencies.

Reply:
Your right! `SubscriptionManager` is not required, I'll remove it in the new constructor. I left it because I did not want to add more changes to this PR, but since you mention it I'll change it.
Totally agree about the interfaces 👍